src/jit/jiteh.cpp

```diff
@@ -2426,6 +2426,11 @@ bool Compiler::fgNormalizeEHCase2()
                                 // this once per dup.
                                 fgReplaceJumpTarget(predBlock, newTryStart, insertBeforeBlk);
 
+                                // Need to adjust ref counts here since we're retargeting edges.
+                                newTryStart->bbRefs++;
+                                noway_assert(insertBeforeBlk->countOfInEdges() > 0);
```

Comment:
Is EH normalization always performed, or is it disabled in minopts? If the former is true, this should just be an `assert`.